omod/src/main/java/org/openmrs/module/patientflags/extension/html/FlagPatientDashboardHeaderExt.java

```diff
@@ -43,14 +43,9 @@ public String getOverrideContent(String test) {
 		
 		if (!results.isEmpty()) {
 			String content = "<TABLE><TR><TD>";
-			
-			// comment out the header text for now
-			//        + Context.getMessageSourceService().getMessage("patientflags.flagpatientdashboard.header") + " ";
-			
 			for (Flag flag : results) {
-				content = content + "<SPAN " + (flag.getPriority() != null ? flag.getPriority().getStyle() : "") + ">" + flag.getLocalizedMessage() + "</SPAN>, ";
+				content = content + "<SPAN " + (flag.getPriority() != null ? flag.getPriority().getStyle() : "") + ">" + flag.getName() + "</SPAN>, ";

```

Comment:
The reason for doing so was to avoid repeated view in patient dashboard. see the image https://cloud.githubusercontent.com/assets/4829880/6397591/9660a760-be07-11e4-979f-a4bf784d84b6.png .
The flags are listed (comma-separated) in dashboard header and in a separate view in dashboard-views too. The idea was to just list flags applicable for patient in main header and list detailed message in a patient-flags overview.